product_dimension/readme/HISTORY.rst

```diff
@@ -0,0 +1,15 @@
+[ The change log. The goal of this file is to help readers
```

Comment:
I think this file can be deleted

Reply:
@emagdalenaC2i readme fragments should never be deleted.

@FrancoMaxime I think people that wrote that file forgot to remove unnecessary comments

Reply:
The history file has been updated.  And a new run of oca-gen-addon-readme has been made.